packages/ListBox/src/components/Content/Content.js

```diff
@@ -54,27 +41,31 @@ const handleContentFocusChange = (hasFocus, dispatch) =>
   dispatch({ type: useListBox.types.setListBoxHasFocus, payload: { hasFocus } });
 
 export default function Content(props) {
-  const { children, hasOptions } = props;
+  const { children, hasOptions, onCancelFooter, ...moreProps } = props;
   const onChangeContext = React.useContext(OnChangeContext);
   const [state, dispatch] = useListBox();
   const { refListBoxContainer } = state;
+  const providedProps = React.useContext(PropsContext);
```

Comment:
I'm not 100% convinced about this name, `providedProps`.  They are the "controlled" props supplied by the consumer that do not affect the internal state of the component (selection, search filter, open state, etc).  

They are "provided" to `ListBox` subcomponents via context for convenience.

Reply:
I think is fine :)